docs/admin-guide/install-buildout.md

```diff
@@ -47,15 +47,17 @@ cd <my_projects>/plone
 Create a Python virtual environment.
 
 ```shell
-python3 -m venv .
+python3 -m venv venv
```

Comment:
Why? This requires more typing when running commands.

Reply:
I can see how it makes it harder to delete and rebuild the virtualenv. I agree with recommending a separate venv folder.